src/platform/baytrail/lib/clk.c

```diff
@@ -106,6 +107,7 @@ static SHARED_DATA struct clock_info platform_clocks_info[] = {
 		.notification_id = NOTIFIER_ID_SSP_FREQ,
 		.notification_mask = NOTIFIER_TARGET_CORE_ALL_MASK,
 		.set_freq = clock_platform_set_ssp_freq,
+		.low_power_mode = NULL,
```

Comment:
Not needed, compiler will do this for you.

Reply:
deleted